smdebug_rulesconfig/profiler_rules/rules.py

```diff
@@ -35,12 +41,14 @@ def __init__(
         :param window: window size for computing quantiles.
         :param scan_interval_us: interval with which timeline files are scanned. Default is 60000000 us.
         """
-        validate_percentile("cpu_threshold_p95", cpu_threshold_p95)
-        validate_percentile("gpu_threshold_p95", gpu_threshold_p95)
-        validate_percentile("gpu_memory_threshold_p95", gpu_memory_threshold_p95)
-        validate_positive_integer("patience", patience)
-        validate_positive_integer("window", window)
-        validate_positive_integer("scan_interval_us", scan_interval_us)
+        validate_percentile(self.__class__.__name__, "cpu_threshold_p95", cpu_threshold_p95)
+        validate_percentile(self.__class__.__name__, "gpu_threshold_p95", gpu_threshold_p95)
+        validate_percentile(
+            self.__class__.__name__, "gpu_memory_threshold_p95", gpu_memory_threshold_p95
+        )
+        validate_positive_integer(self.__class__.__name__, "patience", patience)
+        validate_positive_integer(self.__class__.__name__, "window", window)
+        validate_positive_integer(self.__class__.__name__, "scan_interval_us", scan_interval_us)
```

Comment:
Can you confirm that this whole block is limited to a style change?

Reply:
This isn't a style change, the class is being passed to validation function so that the error message contains the class where validation failed.